app/src/main/res/layout-sw600dp-land/question_player_selection_interaction_item.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should be removed. It was removed in generalisation PR.

Reply:
Removed